apps/transport/test/transport_web/controllers/page_controller_test.exs

```diff
@@ -131,4 +131,10 @@ defmodule TransportWeb.PageControllerTest do
     assert Floki.attribute(item, "href") == ["/login/explanation?redirect_path=%2Finfos_producteurs"]
     assert item |> Floki.text() =~ "Identifiez-vous"
   end
+
+  test "404 page", %{conn: conn} do
+    conn = conn |> get("/notfound")
```

Comment:
Le chemin suivi `/notfound` donne un peu l'impression qu'on demande une page "bien connue" (même si ce n'est pas le cas si j'ai suivi). Peut-être que `/this-page-does-not-exist` serait plus parlant ? (enfin si j'ai compris !)